scripts/commit-and-pr.ps1

```
@@ -0,0 +1,75 @@
+Param(
+    [string]$BranchName = "feature/short-description",
+    [string]$CommitMessage = "",
+    [string]$BaseBranch = "main",
+    [string]$PrTitle = "",
+    [string]$PrBody = ""
+)
+
+Set-StrictMode -Version Latest
+
+function Exec($args) {
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'args' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)